packages/core/i18n/core-i18n-server-internal/src/routes/translations.test.ts

```diff
@@ -12,11 +12,27 @@ import { registerTranslationsRoute } from './translations';
 describe('registerTranslationsRoute', () => {
   test('registers route with expected options', () => {
     const router = mockRouter.create();
-    registerTranslationsRoute(router, 'en');
-    expect(router.get).toHaveBeenCalledTimes(1);
+    registerTranslationsRoute({
+      router,
+      locale: 'en',
+      isDist: true,
```

Comment:
I may have missed it, but is it worth having a jest integration style test for this "isDist" Boolean and how it affects response headers?